desktop/renderer/index.js

```diff
@@ -6,8 +6,6 @@ import {Provider} from 'react-redux'
 import configureStore from '../../react-native/react/store/configure-store'
 import Nav from '../../react-native/react/nav'
 import injectTapEventPlugin from 'react-tap-event-plugin'
-import {reduxDevToolsSelect} from '../../react-native/react/local-debug.desktop'
```

Comment:
redux devtools changed slightly. its faster and better now (ctrl+h to show, ctrl+q to move to various parts of the screen)